commands/browsing/convert-twitter-to-nitter.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+
```

Comment:
According to the CONTRIBUTING.md documentation, in the section [Scripts that require installation of apps](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#scripts-that-require-installation-of-apps), if you need a installation of a third party app in order to run your Script Command, you must inform it at the top of this file the dependency needed. 

In your case, it is this one:
```suggestion
// Dependency: This script requires Nodejs.
// Install Node: https://nodejs.org/en/download/

```